### PR TITLE
Documentation for externalIPs

### PR DESCRIPTION
**Pull request by ArtfulCoder**
#12561

### CODE REVIEW

File: docs/user-guide/services.md [hunk 1]
```diff
@@ -455,6 +455,40 @@ with the user-specified `loadBalancerIP`. If the `loadBalancerIP` field is not s
 an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is specified, but the
 cloud provider does not support the feature, the field will be ignored.
 
+### External IPs
+If there are external IPs that route to one or more cluster nodes, Kubernetes services can be exposed on those
+`externalIPs`. Traffic that ingresses into the cluster with the external IP (as destination IP), on the service port,
+will be routed to one of the service endpoints. `externalIPs` are not managed by Kubernetes and are the responsibility
+of the cluster administrator.
+In the ServiceSpec, `externalIPs` can be specified along with any of the `ServiceTypes`.
```
Comment (thockin) [inline docs/user-guide/services.md]: blank line between pgphs

File: docs/user-guide/services.md [hunk 2]
```diff
@@ -455,6 +455,40 @@ with the user-specified `loadBalancerIP`. If the `loadBalancerIP` field is not s
...
+In the example below, my-service ca be accessed by clients on 80.11.12.10:80 (externalIP:port)
```
Comment (thockin) [inline docs/user-guide/services.md]: s/ca/can

**Comment by thockin** [comment]
nits, then LGTM

File: docs/user-guide/services.md (hunk 2 above)
Comment (ArtfulCoder) [inline docs/user-guide/services.md]: done.

File: docs/user-guide/services.md (hunk 1 above)
Comment (ArtfulCoder) [inline docs/user-guide/services.md]: done

**Comment by thockin** [comment]
LGTM

**Pull request merged**